Scripts/ScriptToSetupAzureADApplicationForAzTSUI.ps1

```diff
@@ -37,6 +37,8 @@ $AzureEnvironmentName = ""
 $AzureEnvironmentAppServiceURI = @{
     "AzureCloud" = "https://{0}.azurewebsites.net";
     "AzureGovernmentCloud" = "https://{0}.azurewebsites.us";
+    "AzureChinaCloud" = "https://{0}.azurewebsites.cn";
```

Comment:
is this correct or this is correct value - https://{0}.chinacloudsites.cn

Reply:
Updated the endpoint to https://{0}.chinacloudsites.cn